frontend/.env.e2e.example

```
@@ -0,0 +1,16 @@
+NEXT_PUBLIC_GRAPHQL_URL=http://localhost:9000/graphql/
+NEXT_PUBLIC_GTM_ID=
+NEXT_PUBLIC_IDX_URL=http://localhost:9000/idx/
+NEXT_PUBLIC_IS_PROJECT_HEALTH_ENABLED=true
+NEXT_PUBLIC_RELEASE_VERSION=
+NEXT_PUBLIC_SENTRY_DSN=
+NEXT_SERVER_CSRF_URL=http://localhost:9000/csrf/
+NEXT_SERVER_DISABLE_SSR=false
+NEXT_SERVER_GITHUB_CLIENT_ID=your-github-client-id
+NEXT_SERVER_GITHUB_CLIENT_SECRET=your-github-client-secret
+NEXT_SERVER_GRAPHQL_URL=http://localhost:9000/graphql/
+NEXTAUTH_SECRET=<your-nextauth-secret>
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXTAUTH_SECRET key should go before the NEXT_PUBLIC_API_URL key

(UnorderedKey)